app/src/main/java/org/schabi/newpipe/MainActivity.java

```diff
@@ -551,8 +561,6 @@ public boolean onCreateOptionsMenu(Menu menu) {
         if (!(fragment instanceof SearchFragment)) {
             findViewById(R.id.toolbar).findViewById(R.id.toolbar_search_container).setVisibility(View.GONE);
 
-            MenuInflater inflater = getMenuInflater();
-            inflater.inflate(R.menu.main_menu, menu);
```

Comment:
Are we sure we want to completely remove the main menu?
I think this is ok, but since you removed the menu inflation, you should also remove the corresponding menu layout xml file.

Reply:
Yes. This menu won't be required any more in this activity. I removed main_menu.xml in b965f88e. thanks;)